packages/backend-for-frontend/src/api/catalogApiConverter.ts

```diff
@@ -61,6 +61,7 @@ export function toBffCatalogApiEService(
       activeDescriptor,
       requesterTenant
     ),
+    isDelegable: eservice.isDelegable,
```

Comment:
See the comment above, it should not be added to this model (a list compact model) but to the ones that don't return compact stuff